docs/access-jh.md

```diff
@@ -1,6 +1,6 @@
 # Access JupyterHub Environment
 
-* In order to access the environment for the development of the project, you will have to be added to the odh-env-users group [here](https://github.com/orgs/os-climate/teams/odh-env-users). This can be done by opening an issue on [this page](https://github.com/os-climate/aicoe-osc-demo/issues) with the title "Add <USERNAME> to odh-env-users".
+* In order to access the environment for the development of the project, you will have to be added to the odh-env-users group [here](https://github.com/orgs/os-climate/teams/odh-env-users). This can be done by opening an issue on [this page](https://github.com/os-climate/OS-Climate-Community-Hub/issues/new) with the title "Add <USERNAME> to odh-env-users".
```

Comment:
Hi @Shreyanand ,

The link "https://github.com/os-climate/OS-Climate-Community-Hub/issues/new" takes you to a blank issue page. As we already have an issue templete for onboarding, do you think the link should be directed to these issue templetes? (https://github.com/os-climate/OS-Climate-Community-Hub/issues/new/choose)

Reply:
currently we don't have a template for adding to the odh-env-users, a github team that once you're added to, you can access the JH environment. The instructions in the readme specify what to put in the issue. Therefore it made sense to link to a new blank issue.

Reply:
Oh Ok. Sounds good!! Else /lgtm